src/TicketingSystem.DatabaseInitializationApp/Program.cs

```diff
@@ -0,0 +1,253 @@
+﻿using Microsoft.Extensions.Configuration;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Threading.Tasks;
+using TicketingSystem.DataAccess.Entities;
+using TicketingSystem.DataAccess.Enums;
+using TicketingSystem.DataAccess.Repositories;
+
+namespace TicketingSystem.DatabaseInitializationApp
+{
+    public class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var config = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("settings.json")
```

Comment:
I see - setting.json is in git.ignore file, and settings.example.json should be used here. Can we resolve this naming programmatically - through the development settings or just regex to make it more convenient for reviewers to run your code locally with minimal changes.

Reply:
Fixed